### ISSUE TITLE
Error during install: 'ndx-pose' distribution was not found

### ISSUE THREAD

**Issue by Deruijter** [talmolab/sleap#859]
I am currently newly installing sleap on our system using the instructions on the website.
I tried both: 
`conda create -y -n sleap -c sleap -c nvidia -c conda-forge sleap`
and
`conda create -y -n sleap -c sleap -c nvidia -c conda-forge sleap=1.2.5`

Subsequently I run:
```
conda activate sleap
sleap-label
```

Then an error appears:
```
(sleap) C:\Users\xxxxxxx>sleap-label
Traceback (most recent call last):
  File "C:\Users\xxxxxxx\.conda\envs\sleap\Scripts\sleap-label-script.py", line 33, in <module>
    sys.exit(load_entry_point('sleap==1.2.5', 'console_scripts', 'sleap-label')())
  File "C:\Users\xxxxxxx\.conda\envs\sleap\Scripts\sleap-label-script.py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\importlib_metadata\__init__.py", line 194, in load
    module = import_module(match.group('module'))
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\sleap\__init__.py", line 10, in <module>
    from sleap.io.dataset import Labels, load_file
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\sleap\io\dataset.py", line 70, in <module>
    from sleap.instance import (
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\sleap\instance.py", line 32, in <module>
    from sleap.io.video import Video
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\sleap\io\video.py", line 17, in <module>
    from sleap.util import json_loads, json_dumps
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\sleap\util.py", line 11, in <module>
    from pkg_resources import Requirement, resource_filename
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\pkg_resources\__init__.py", line 3266, in <module>
    @_call_aside
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\pkg_resources\__init__.py", line 3241, in _call_aside
    f(*args, **kwargs)
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\pkg_resources\__init__.py", line 3279, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\pkg_resources\__init__.py", line 573, in _build_master
    ws.require(__requires__)
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\pkg_resources\__init__.py", line 891, in require
    needed = self.resolve(parse_requirements(requirements))
  File "C:\Users\xxxxxxx\.conda\envs\sleap\lib\site-packages\pkg_resources\__init__.py", line 777, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'ndx-pose' distribution was not found and is required by sleap
```

And the application does not start.

I am using windows 10 with python 3.7.12 installed.

The problem is solved after manually installing ndx-pose using `pip install ndx-pose`

**Comment by talmo** [comment]
O shoot, sorry about that @Deruijter -- we neglected to include `ndx-pose` in our conda packages. Thanks for reporting this and yes, the workaround for now is to manually `pip install ndx-pose` after the conda installation. We'll put out a fix ASAP.

**Comment by talmo** [comment]
Hi @Deruijter, this is now fixed in [SLEAP v1.2.6](https://github.com/talmolab/sleap/releases/tag/v1.2.6).

Thanks for bringing it to our attention!! Let us know if you run into any other issues.

Cheers,

Talmo
